docs/data/data-grid/rows/rows.md

```diff
@@ -80,6 +82,20 @@ In addition, the area in which `onRowsScrollEnd` is called can be changed using
 
 {{"demo": "InfiniteLoadingGrid.js", "bg": "inline", "disableAd": true}}
 
+### Lazy loading [<span class="plan-pro"></span>](https://mui.com/store/items/mui-x-pro/)
+
+By default, infinite loading works on the client-side.
```

Comment:
I would not talk about *infinite loading* which is mostly use to describe the scenario where the scrollbar shrink as you scroll (not what we have here if I understand correctly, the grid does not support infinite loading at all).


The problem might come from the name of `useGridInfiniteLoader` which is, I think mis-named